Dockerfile

```diff
@@ -53,8 +53,8 @@ COPY . /squad-build/
 ENV SQUAD_STATIC_DIR=/app/static
 
 RUN cd /squad-build && ./scripts/git-build && \
-    pip3 install --no-dependencies ./dist/squad*.whl squad-linaro-plugins sentry-sdk==0.14.3 && \
-    pip3 install --no-dependencies ./dist/squad*.whl zipp importlib-metadata asgiref django-bootstrap3 && \
+    pip3 install --no-dependencies ./dist/squad*.whl squad-linaro-plugins sentry-sdk==0.14.3 zipp importlib-metadata asgiref django-bootstrap3 django-storages && \
```

Comment:
Can you make it `django-storages==1.9`? This will support `python3-django` (Django 1.11)

Reply:
OK